cmd/kyma/alpha/create/module/module.go

```diff
@@ -407,6 +407,12 @@ func (cmd *command) Run(ctx context.Context) error {
 			annotations["operator.kyma-project.io/is-cluster-scoped"] = "false"
 		}
 
+		version := cmd.opts.Version
+		if modCnf != nil {
+			version = modCnf.Version
+		}
+		annotations["operator.kyma-project.io/module-version"] = version
```

Comment:
If you have time, this whole annotations and label part would be a nice candidate to extract and make this crazy long function body shorter. Also possible to make it nicer in terms of using consts for labels, etc. WDYT?

Reply:
I agree specifically with having the labels and annotations as const :D